components/Image/Image.tsx

```diff
@@ -19,12 +31,24 @@ const loader: ImageLoader = ( { quality, src, width } ) => {
 	return src;
 }
 
-export default function Image ( props: Props ) {
+export default function Image ( props: Type ) {
+	const imageProps = {
+		alt: props.alt || '',
+		srcSet: props.srcset || undefined,
+		src: props.src,
+		width: props.width || props.originalWidth,
+		height: props.height || props.originalHeight,
+		layout: props.width ? 'fixed' as any : 'responsive' as any,
+	};
```

Comment:
this is probably better expressed as an override:
```suggestion
	const imageProps = {
		...props,
		srcSet: props.srcset || undefined,
		src: props.src,
		width: props.width || props.originalWidth,
		height: props.height || props.originalHeight,
		layout: props.width ? 'fixed' as any : 'responsive' as any,
	};
```